canopy.config.js

```diff
@@ -20,6 +20,18 @@ exports.dev = {
     "https://canopy-iiif.vercel.app/iiif/manifest/nez-perce/02-lawyer.json",
   ],
   metadata: ["Date", "Subject"],
+  seo: {
+    hostname: "https://canopy-iiif.vercel.app",
+    description: "",
+    analytics: "",
+    metatags: {
+      googleSiteVerification: "",
+      twitter: {
+        site: "",
+        handle: "",
+      }
+    }
+  }
```

Comment:
Really like this approach.

Reply:
@mathewjordan 

1. Should we set defaults for as much as possible (Twitter site, Twitter handle, etc.). If so, what should those defaults be?
2. Should description be a thing or should we pull that from the collection manifest?  I just checked, and while this is defined, the property is never actually used.
3. Host name is similar.  Initially, I was thinking of defining this here for the purpose of canonical links, but I didn't actually implement.  I'm thinking I may just drop both props here.